redis/commands/search/field.py

```diff
@@ -106,13 +110,21 @@ class TagField(Field):
 
     SEPARATOR = "SEPARATOR"
     CASESENSITIVE = "CASESENSITIVE"
+    WITHSUFFIXTRIE = "WITHSUFFIXTRIE"
```

Comment:
Maybe these args could be somewhere else, but referred to? Looks like duplication